php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1398,6 +1398,23 @@ class NDB_BVL_Instrument extends NDB_Page
         return false;
     }
 
+    /**
+     * Gets the VisitLabel for the timepoint to which this instrument pertains
+     *
+     * @return String for the VisitLabel
+     */
+    function getVisitLabel()
+    {
+        $sessionID = $this->getSessionID();
+
+        if (!empty($sessionID)) {
+            $t          = Timepoint::singleton($sessionID);
```

Comment:
i am wondering if this should be namespaced? i.e. \Timepoint::singleton($sessionID)?